index.html

```diff
@@ -106,37 +106,47 @@
 <body>
   <article>
     <header>
-      This is a clone 2048 game
+      <h1>
+        This is a clone 2048 game
+      </h1>
     </header>
     <section id="main" class="grid-3x3">
       <div class="top-l"></div>
       <div class="top-m">
-        <button type="button" class="swipe swipe-u" id="swipe-up" value="up"></button>
+        <button type="button" class="btn swipe swipe-u js-swipe-btn" id="swipe-up" value="up"></button>
       </div>
       <div class="top-r"></div>
       <div class="mid-l">
         <!-- BOARD goes here -->
-        <button type="button" class="swipe swipe-l" id="swipe-left" value="left"></button>
+        <button type="button" class="btn swipe swipe-l js-swipe-btn" id="swipe-left" value="left"></button>
       </div>
       <div class="mid-m">
         <div id="board" class="board"></div>
       </div>
       <div class="mid-r">
-        <button type="button" class="swipe swipe-r" id="swipe-right" value="right"></button>
+        <button type="button" class="btn swipe swipe-r js-swipe-btn" id="swipe-right" value="right"></button>
       </div>
       <div class="bot-l"></div>
       <div class="bot-m">
-        <button type="button" class="swipe swipe-d" id="swipe-down" value="down"></button>
+        <button type="button" class="btn swipe swipe-d js-swipe-btn" id="swipe-down" value="down"></button>
       </div>
       <div class="bot-r"></div>
+      <!-- https://developer.mozilla.org/en-US/docs/Web/Web_Components -->
+      <custom-modal title="You fucking lost, idiot." description="Maybe don't suck so much? ¯\(ツ)/¯" class="js-modal-lost">
```

Comment:
my life would probably be better had I been told this more as a child. Also, where does the `description` come into play?

Reply:
`title` and `description` are effectively props, like React, and should be rendering in the modal. currently the styling for `<p>` doesn't actually have the tag showing in the modal tho for whatever reason